builtin/credential/ldap/backend.go

```diff
@@ -195,7 +195,11 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	// Policies from each group may overlap
 	policies = strutil.RemoveDuplicates(policies, true)
 
-	entityAliasAttribute, err := ldapClient.GetUserAliasAttributeValue(cfg.ConfigEntry, c, username)
+	if usernameAsAlias {
+		return username, policies, ldapResponse, allGroups, nil
+	}
+	
+	entityAliasAttribute, err = ldapClient.GetUserAliasAttributeValue(cfg.ConfigEntry, c, username)
```

Comment:
Undefined error here because `:=` is missing.

Reply:
Yes, I did not check the previous commit enough sorry about that. It should be good now.